qiskit/providers/aer/extensions/snapshot_statevector.py

```diff
@@ -21,12 +21,14 @@
 class SnapshotStatevector(Snapshot):
     """ Snapshot instruction for statevector snapshot type """
 
-    def __init__(self, label, num_qubits=0):
+    def __init__(self, label, num_qubits=0, dict_form=False):
```

Comment:
I think this can be removed, not worth adding this to a function thats about to be deprecated